go/vt/wrangler/keyspace.go

```diff
@@ -897,6 +897,11 @@ func (wr *Wrangler) DeleteKeyspace(ctx context.Context, keyspace string, recursi
 		}
 	}
 
+	// Delete the cell-global VSchema path
+	// If not remove this, vtctld web page Dashboard will Display Error
+	vschema := &vschemapb.Keyspace{}
+	wr.ts.SaveVSchema(ctx, keyspace, vschema)
```

Comment:
You probably want to catch the error here:

if err := wr.ts.SaveVSchema(ctx, keyspace, vschema); err != nil && err != topo.ErrNoNode {
    return err
}

or something similar....